_layouts/post.html

```diff
@@ -1,10 +1,22 @@
 ---
 layout: default
+image:
+  path: /img/posts/{{page.img}}"
+  height: 100
+  width: 100
 ---
 
 <article class="post-container">
+  <meta property="twitter:image" content= "/img/posts/{{page.img}}" >
+  <meta property="og:image" content= "https://teachla.uclaacm.com/img/posts/default.png" >
     <h1 class="title page-title">{{page.title}}</h1>
+    <!-- required for Facebook "Share" button -->
+    <div id="fb-root"></div>
+    <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v9.0" nonce="VotTmBal"></script>
+
+
```

Comment:
From the FB documentation:
>  Include the JavaScript SDK on your page once, ideally right after the opening body tag.

We want the script tag to load as quickly as possible, since we're making the request in the background as the rest of the document/tags are parsed. 
* at the very least, we should move it to the top of the layout
* something you can consider: maybe modifying `_layout/default.html` to conditionally load this, but that's overkill for now probably